app/style/components/service-avatar.less

```diff
@@ -17,7 +17,8 @@
  *
  */
 
-service-avatar {
+service-avatar,
+participant-avatar {
```

Comment:
Let's cleanup the CSS classes and remove the outdated service and user avatars in the next step

Reply:
Most definitely!